sci-libs/gdal/gdal-2.0.2-r1.ebuild

```diff
@@ -0,0 +1,282 @@
+	sed -i \
+		-e "s:library_dirs = :library_dirs = /usr/$(get_libdir):g" \
+		swig/python/setup.cfg || die "sed python setup.cfg failed"
+
+	tc-export AR RANLIB
+
+	eautoreconf
+}
+
+src_configure() {
+	local myopts=""
+
+	if use java; then
+		myopts+="
+			--with-java=$(java-config --jdk-home 2>/dev/null)
+			$(use_with mdb)"
+	else
+		myopts+=" --without-java --without-mdb"
+		use mdb && ewarn "mdb requires java use enabled. disabling"
+	fi
+
+	if use sqlite; then
+		myopts+=" LIBS=-lsqlite3"
+	fi
+
+	# pcidsk is internal, because there is no such library yet released
+	#     also that thing is developed by the gdal people
+	# kakadu, mrsid jp2mrsid - another jpeg2k stuff, ignore
+	# bsb - legal issues
+	# oracle - disabled, i dont have and can't test
+	# ingres - same story as oracle oci
+	# podofo - we use poppler instead they are exclusive for each other
+	# tiff is a hard dep
+	ECONF_SOURCE="${S}" econf \
+		--includedir="${EPREFIX}/usr/include/${PN}" \
+		--disable-static \
+		--enable-shared \
+		--with-expat \
+		--with-geotiff \
+		--with-grib \
+		--with-libtiff \
+		--with-libz="${EPREFIX}/usr/" \
+		--with-ogr \
+		--without-bsb \
+		--without-dods-root \
+		--without-dwgdirect \
+		--without-epsilon \
+		--without-fme \
+		--without-grass \
+		--without-hdf4 \
+		--without-idb \
+		--without-ingres \
+		--without-jp2mrsid \
+		--without-kakadu \
+		--without-libtool \
+		--without-mrsid \
+		--without-msg \
+		--without-oci \
+		--without-pcraster \
+		--without-podofo \
+		--without-python \
+		--without-sde \
+		$(use_enable debug) \
+		$(use_with armadillo) \
+		$(use_with aux_xml pam) \
+		$(use_with curl) \
+		--without-ecw \
+		$(use_with fits cfitsio) \
+		$(use_with geos) \
+		$(use_with gif) \
+		$(use_with gml xerces) \
+		$(use_with hdf5) \
+		$(use_with jpeg pcidsk) \
+		$(use_with jpeg) \
+		$(use_with jpeg2k jasper) \
+		$(use_with mysql mysql "${EPREFIX}"/usr/bin/mysql_config) \
+		$(use_with netcdf) \
+		$(use_with odbc) \
+		$(use_with ogdi ogdi "${EPREFIX}"/usr) \
+		$(use_with opencl) \
+		$(use_with pdf poppler) \
+		$(use_with perl) \
+		$(use_with png) \
+		$(use_with postgres pg) \
+		$(use_with spatialite) \
+		$(use_with sqlite sqlite3 "${EPREFIX}"/usr) \
+		$(use_with threads) \
+		$(use_with xls freexl) \
+		${myopts}
+
+	# mysql-config puts this in (and boy is it a PITA to get it out)
+	if use mysql; then
+		sed -i \
+			-e "s: -rdynamic : :" \
+			GDALmake.opt || die "sed LIBS failed"
+	fi
+}
+
+src_compile() {
+	if use perl; then
+		rm "${S}"/swig/perl/*_wrap.cpp || die
+		emake -C "${S}"/swig/perl generate
+	fi
+
+	# gdal-config needed before generating Python bindings
+	default
+
+	if use perl ; then
+		pushd "${S}"/swig/perl > /dev/null
+		perl-module_src_configure
+		perl-module_src_compile
+		popd > /dev/null
+	fi
+
+	if use python; then
+		rm -f "${S}"swig/python/*_wrap.cpp || die
+		emake -C "${S}"/swig/python generate
+		pushd "${S}"/swig/python > /dev/null
```

Comment:
die's added. Not sure about a separate commit as it is a new file anyway - would do the old ones after adding the new stuff?